packages/cells/src/widget.ts

```diff
@@ -1574,6 +1581,13 @@ export class MarkdownCell extends AttachmentsCell<IMarkdownCellModel> {
     }
   }
 
+  set showEditorForReadOnlyMarkdown(value: boolean) {
```

Comment:
Could you either use a method `setShowEditorForReadOnly` or a getter? And add a small doc string.

Defining only a setter is not a good practice because it feels strange to be able to do `myObj.a = value` but not `const value = myObj.a`.

> It is ok to drop `Markdown` as we are in the `MarkdownCell`